### PR TITLE
More SpineOpt

### PR DESCRIPTION
**Pull request by joaquimg**
- Add info in main readme
- Case simplified to run a single month (instead of 12)
- @datejada there are still some non-smooth install issues: https://github.com/spine-tools/SpineOpt.jl/issues/1142

### CODE REVIEW

**Comment by joaquimg** [comment]
There is something non-deterministic.
Every time I run I get a different Hash.
Is this expected @datejada ?

**Comment by joaquimg** [comment]
Row labels get messed up:
image

**Comment by datejada** [comment]
> There is something non-deterministic. Every time I run I get a different Hash. Is this expected @datejada ?

@joaquimg I'm not aware that something is non-deterministic. @manuelma or @tasqu, do you have any idea about this behaviour in SpineOpt? Basically, the order of the constraints in the .mps file seems to change for each run.

**Comment by joaquimg** [comment]
@odow, can you test running the SpineOpt script on your side to see if everything is properly documented?

**Comment by Tasqu** [comment]
@datejada I'm not surprised it's non-deterministic. We use Julia `Dict`s in SpineOpt quite a bit and their ordering is not consistent. Furthermore, I talked with Antti, and depending on how SpineInterface queries the databases it might also go through Python dictionaries which are not ordered either.

File: README.md [hunk 1]
```diff
@@ -94,6 +94,27 @@ julia --project=Sienna Sienna/tutorial_1.jl --case=PTDF-12 --run [--write]
 
 See the `Sienna/tutorial_1.jl` driver script for more details.
 
+### SpineOpt
+
+For now, we can rebuild all of the SpineOpt examples with (from the
+root of this directory):
+
+```
```
Comment (blegat) [inline README.md]:
```suggestion
```sh
```

File: README.md [hunk 2]
```diff
@@ -94,6 +94,27 @@ julia --project=Sienna Sienna/tutorial_1.jl --case=PTDF-12 --run [--write]
...
+make spineopt
+```
+
+To run a particular case, do:
+
+```
```
Comment (blegat) [inline README.md]:
```suggestion
```sh
```

**Comment by joaquimg** [comment]
I am merging this, as it is a strict improvement over what we currently have.

**Pull request merged**